examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonDataFrameExample.scala

```diff
@@ -54,13 +54,13 @@ object CarbonDataFrameExample {
     // Saves dataframe to carbondata file
     df.write
       .format("carbondata")
-      .option("tableName", "carbon_table")
+      .option("tableName", "carbon_df_table")
```

Comment:
why requires to rename table name, is there any rule?

Reply:
I have just changed because of creating fails sometimes because all other examples use the same name and also better differentiate the table name as per example.